robottelo/ui/navigator.py

```diff
@@ -20,7 +21,14 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
-        self.browser.refresh()
+        # 1328627 - as the number of subs increases, dashboard page gets slower
```

Comment:
Hmm, why do we have reference to dashboard page at all? It is not a default sat page since 6.2